builder/vmware/common/step_clean_vmx.go

```diff
@@ -59,8 +60,10 @@ func (s StepCleanVMX) Run(state multistep.StateBag) multistep.StepAction {
 		vmxData[ide+"clientdevice"] = "TRUE"
 	}
 
-	ui.Message("Disabling VNC server...")
-	vmxData["remotedisplay.vnc.enabled"] = "FALSE"
+	if !s.SkipVNCDisable {
```

Comment:
Not a fan of the triple negative in the concept of "not skip disable" from a readability perspective. Could we call this option something that involves less mental gymnastics?  Why not VNCDisable instead?  and then check against it to decide to skip? Or just EnableVNC instead, and have it be all positives.